master/internal/task/allocation.go

```diff
@@ -395,6 +395,7 @@ func (a *Allocation) TaskContainerStateChanged(
 	switch msg.Container.State {
 	case cproto.Pulling:
 		a.state = model.MostProgressedAllocationState(a.state, model.AllocationStatePulling)
+		a.model.StartTime = time.Now().UTC().Truncate(time.Millisecond)
```

Comment:
we need to make sure this gets saved immediately, i'm not sure when the next time the model is sync is

Reply:
Got it! I see an UpdateAllocationState method in master/db/database.go but that only updates the allocation state. Should I create a similar method that updates the start time?

Reply:
done. Should I add a check for this to the postgres_task_intg_test.go?